Projects/Claims/Sources/ClaimStore.swift

```diff
@@ -30,13 +30,11 @@ public final class ClaimsStore: StateStore<ClaimsState, ClaimsAction> {
                         callback(.value(ClaimsAction.setClaims(claims: claimData)))
                     }
                     .onError { error in
-                        if ApplicationContext.shared.isDemoMode {
-                            callback(.value(.setLoadingState(action: action, state: nil)))
-                        } else {
-                            callback(
-                                .value(.setLoadingState(action: action, state: .error(error: L10n.General.errorBody)))
+                        callback(
```

Comment:
Why did we check for demo mode here?

Reply:
Because we dont want to show error if this fail for demo mode and it will since we dont have proper token

Reply:
ah I see